lnbits/decorators.py

```diff
@@ -158,6 +158,19 @@ async def check_user_exists(
     return user
 
 
+async def authenticated_user_id(
+    access_token: Annotated[Optional[str], Depends(check_access_token)],
+    usr: Optional[UUID4] = None,
+) -> Optional[str]:
+    if usr and settings.is_auth_method_allowed(AuthMethods.user_id_only):
+        return usr.hex
+    if access_token:
+        account = await _get_account_from_token(access_token)
```

Comment:
hm i think this should always return an account or raise an exception

Reply:
why?

Reply:
renamed to `optional_user_id` to avoid confusion